djangocms_versioning_filer/monkeypatch/admin/settings.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+
+import filer
+
+
+FILER_FILE_CONSTRAINTS = getattr(
```

Comment:
Move out to aldryn_config.py (append to FILER_FILE_CONSTRAINTS if that values doesn't already exists in it; create FILER_FILE_CONSTRAINTS if it doesn't exist as well)